tests/scenarios/test_ads_search_telemetry.py

```diff
@@ -156,20 +147,27 @@ def wait_for_ad_click_page_to_load(selenium):
 
 @then("The user goes back to the search page")
 def go_back_one_page(selenium):
+    url = selenium.current_url
     selenium.back()
-    time.sleep(10)  # wait some time after going back so the event can register
+    # wait some time after going back so the event can register
+    WebDriverWait(selenium, 30).until(EC.url_changes(url))
 
 
-@then("The user searches for something on Google")
-def load_and_search_on_google(selenium):
-    url = "http://www.google.com"
+@then("The user searches for something")
+def load_and_search_on_mozsearch(selenium, search_server):
+    url = f"{search_server}/searchTelemetryAd.html?s=test"
     selenium.get(url)
-    text_box = selenium.find_element(By.CSS_SELECTOR, "form textarea")
-    text_box.send_keys("buy stocks", Keys.ENTER)
-    WebDriverWait(selenium, 60).until(EC.url_changes(url))
+
+
+@then("The user clicks on an ad")
+def click_on_ad_local_search(selenium):
+    time.sleep(10)
```

Comment:
Does this need to sleep for _10_ seconds?

Reply:
Yes, there needs to be some time for the network traffic to be inspected by the search telemetry code